app/controllers/duties_controller.rb

```diff
@@ -88,6 +92,17 @@ def grabable?(duty_id_params)
     end
   end
 
+  def drop_eligible?(drop_duty_ids)
```

Comment:
I think better method name is `can_drop_duties?`

Reply:
Okay, will be changed in the next commit.